src/themes/default/components/core/blocks/Form/BaseInputNumber.vue

```diff
@@ -11,6 +11,22 @@
       @input="$emit('input', $event.target.value)"
       @blur="$emit('blur', $event.target.value)"
     >
+    <template v-if="validation">
```

Comment:
i see that it's the same code as in BaseInput - so lets make a component from that :)
phrase `<template v-if="validation">` appears in four input component so it's big code redundancy.

And here (and other components) we can just have a slot for this new validation component.